src/pages/groupComparison/GroupComparisonPage.tsx

```diff
@@ -65,7 +73,14 @@ export default class GroupComparisonPage extends React.Component<
     @observable.ref private store: GroupComparisonStore;
     private queryReaction: IReactionDisposer;
     private urlWrapper: GroupComparisonURLWrapper;
-
+    clearMessage: (() => void) | undefined;
+    currentPathway: string = '';
+    rankingChoices: PMParameters = {
```

Comment:
These seem specific to pathway mapper and should probably not be up in the GroupComparison page unless they need to be shared with other tabs.  The names are too generic.  clearMessage?  Clear what message?

Reply:
I think the reason why we have these props in GroupComparisonPage component is to persist user selection. Every time user selects/deselects a group on the top of the page, all the tabs within the page gets re-rendered and we lose the user selection like selected pathway.

We should investigate further to see if persisting the user selection is possible within the Pathways component itself. It would be nice if we can keep these pathway mapper specific params limited to the Patways component only.